src/extension/src/__main__.py

```diff
@@ -46,6 +48,8 @@ def main(argv):
         runtime_context_handler = RuntimeContextHandler(logger)
         json_file_handler = JsonFileHandler(logger)
         ext_env_handler = ExtEnvHandler(json_file_handler)
+        env_layer = EnvLayer()
+        env_manager = EnvManager(env_layer)
```

Comment:
Ensure there is a backlog item to convert all this into dependency injection like Core does it.

Reply:
Added task# 10114420